unlock-app/src/queries/keyholdersByLock.js

```diff
@@ -13,6 +13,7 @@ export default function keyholdersByLockQuery() {
         }
         name
         address
+        owner
```

Comment:
This is used to only ask the user to check their wallet if they are the lock owner.